charts/turbinia/README.md

```diff
@@ -0,0 +1,322 @@
+<!--- app-name: Turbinia -->
+# Turbinia Helm Chart
+
+Turbinia is an open-source framework for deploying, managing, and running distributed forensic workloads. 
+
+[Overview of Turbinia](https://turbinia.readthedocs.io/en/latest/)
+
+[Chart Source Code](https://github.com/google/osdfir-infrastructure)
+## TL;DR
+
+```console
+helm install turbinia-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/turbinia
+```
+> **Tip**: To quickly get started with a local cluster, see [minikube install docs](https://minikube.sigs.k8s.io/docs/start/).
+
+## Introduction
+
+This chart bootstraps a [Turbinia](https://github.com/google/turbinia/blob/master/docker/release/build/Dockerfile-latest) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+
+> **Note**: Currently Turbinia only supports processing of GCP Persistent Disks and Local Evidence. See [GKE Installations](#gke-installations) for deploying to GKE.
+
+## Installing the Chart
+
+To install the chart, specify any release name of your choice. For example, if you
+want to install the chart for development, you can choose a release name of `turbinia-dev` then run:
+```console
+helm install turbinia-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/turbinia
```

Comment:
Should this be with `turbinia-dev` to match above?

Reply:
Good catch, fixed